src/webrtc.ts

```diff
@@ -98,13 +107,26 @@ export class WebrtcExtension extends EventEmitter {
   prepareSession (options: any) {
     options.config.iceServers = this.config.iceServers || options.iceServers;
     options.config.iceTransportPolicy = this.config.iceTransportPolicy || 'all';
-    options.optOutOfWebrtcStatsTelemetry = !!this.config.optOutOfWebrtcStatsTelemetry;
-
-    const session = new GenesysCloudMediaSession(options, this.getSessionTypeByJid(options.peerID), this.config.allowIPv6);
+    options.optOutOfWebrtcStatsTelemetry = !!this.config
+      .optOutOfWebrtcStatsTelemetry;
+
+    const session = new GenesysCloudMediaSession(
+      options,
+      this.getSessionTypeByJid(options.peerID),
+      this.config.allowIPv6
+    );
     this.proxyStatsForSession(session);
     return session;
   }
 
+  calculatePayloadSize (trace: any): number {
```

Comment:
NitPick: I don't think this really belongs as a class fn. It's probably better suited as an exported util. I'd put it in `utils.ts`.

Reply:
Moved it.